test/lib/deployment/test-deployment.js

```diff
@@ -449,6 +452,8 @@ async function testDeployment(fixturePath, opts = {}) {
         throw err;
       }
 
+      const retryDelay = Math.max(probe.retryDelay || 0, err.retryDelay || 0);
```

Comment:
Does this implicitly assume that `probe` and `err` will have different delays and won't step on each other? @EndangeredMassa